WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -728,21 +737,43 @@ private extension ProductsViewController {
     ///
     func displayNoResultsOverlay() {
         let emptyStateViewController = EmptyStateViewController(style: .list)
+        let config = filters.numberOfActiveFilters == 0 ? createNoProductsConfig() : createNoProductsMatchFilterConfig()
```

Comment:
This is a bit of a personal preference, but I don't find these kind of assignments too easy to parse. I would suggest encapsulating if in a method, so that, here, the code could read like `let config = filterConfiguration()` or similar, hiding the ternary operator (which, I know from my own experience, is the opposite we do on the Android codebase)

Reply:
Yeah, I see your point and I agree, fixed in https://github.com/woocommerce/woocommerce-ios/pull/5372/commits/a2e64927c12537356911ed797c80001717c8e8cc